jest.config.json

```diff
@@ -12,7 +12,9 @@
   "clearMocks": true,
   "testMatch": ["<rootDir>/tests/**/*.(test).ts(x)?"],
   "moduleNameMapper": {
-    "\\.(css|scss|less)$": "<rootDir>/tests/__utils__/style-mock.ts"
+    "\\.(css|scss|less)$": "<rootDir>/tests/__utils__/style-mock.ts",
+    "$comment": "react-markdown throws syntax errors (ESM-related). This imports the .js file directory without issue",
```

Comment:
the comment shouldn't be an entry in `moduleNameMapper`. Can you update it to be:
`\\react-markdown throws syntax err...`